libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/component/camera/CameraViewModel.kt

```diff
@@ -1,14 +1,10 @@
 package com.mapbox.navigation.dropin.component.camera
 
 import android.location.Location
-import androidx.lifecycle.asFlow
 import com.mapbox.geojson.Point
 import com.mapbox.maps.CameraOptions
 import com.mapbox.navigation.core.MapboxNavigation
-import com.mapbox.navigation.dropin.component.location.LocationBehavior
 import com.mapbox.navigation.dropin.lifecycle.UIViewModel
-import kotlinx.coroutines.flow.collect
-import kotlinx.coroutines.launch
 
 sealed class CameraAction {
     data class UpdateLocation(val location: Location) : CameraAction()
```

Comment:
We can delete this action.

Reply:
As per internal discussion, this will be fixed in Camera when we start fixing it.